src/corehost/cli/runtime_config.cpp

```diff
@@ -112,3 +112,18 @@ bool runtime_config_t::get_portable() const
 {
     return m_portable;
 }
+
+void runtime_config_t::config_kv(std::vector<std::string>* keys, std::vector<std::string>* values) const
+{
+    for (const auto& kv : m_properties)
+    {
+        keys->push_back(pal::to_stdstring(kv.first));
+        values->push_back(pal::to_stdstring(kv.second));
+    }
+
+    if (m_properties.count(_STRINGIFY(System_GC_Server)) == 0)
```

Comment:
This would mean we will need to update the host everytime a new property (e.g. min or max threads) would need to to be added. Is that right? Any reason we cannot just read the values and set them as configuration settings without knowing about them?

CC @sergiy-k @kouvel